etc/openqa/openqa.ini

```diff
@@ -1,9 +1,9 @@
 [global]
 ## Web site name for tab titles and bookmarks
-#appname = openQA
+appname = openQA local okurz
```

Comment:
that's part of the "WIP" commits, just concentrate on the screenshot for now :-)